src/utils/components/Charts/ComponentReady/ComponentReady.tsx

```diff
@@ -7,18 +8,28 @@ import { Bullseye } from '@patternfly/react-core';
 
 type ComponentReadyProps = React.PropsWithChildren<{
   isReady: boolean;
+  linkToMetrics?: string;
   spinner?: boolean;
   text?: string;
 }>;
 
-const ComponentReady: React.FC<ComponentReadyProps> = ({ children, isReady, spinner, text }) => {
+const ComponentReady: React.FC<ComponentReadyProps> = ({
+  children,
+  isReady,
+  linkToMetrics,
+  spinner,
+  text,
+}) => {
   const { t } = useKubevirtTranslation();
   return isReady ? (
     <>{children}</>
   ) : (
     <Bullseye className="ComponentReady">
-      {(!spinner || text) && <MutedTextSpan text={text || t('No data available')} />}
+      {(text || (!spinner && !linkToMetrics)) && (
+        <MutedTextSpan text={text || t('No data available')} />
+      )}
       {spinner && <Loading />}
+      {linkToMetrics && <Link to={linkToMetrics}>{t('No data available')}</Link>}{' '}
```

Comment:
```suggestion
      {(!spinner && !linkToMetrics) && (
        <MutedTextSpan text={renderText} />
      )}
      {spinner && <Loading />}
      {linkToMetrics && <Link to={linkToMetrics}>{renderText}</Link>}
```
check if this is wokring